src/ol/format/kmlformat.js

```diff
@@ -907,6 +918,31 @@ ol.format.KML.readMultiGeometry_ = function(node, objectStack) {
   } else {
     return new ol.geom.GeometryCollection(geometries);
   }
+
+  /**
+   * @param {ol.geom.MultiPoint|ol.geom.MultiLineString|ol.geom.MultiPolygon}
+   *   multiGeometry
+   */
+  function setCommonGeometryProperties(multiGeometry) {
```

Comment:
We do not use such inner functions too much. I'd recommend changing it to a function on the `ol.format.KML` namespace: `ol.format.KML.setCommonGeometryProperties_`.